src/main/scala/com/databricks/spark/redshift/Utils.scala

```diff
@@ -20,10 +20,18 @@ import java.io.File
 import java.net.URI
 import java.util.UUID
 
+import scala.collection.JavaConverters._
+import scala.util.control.NonFatal
+
+import com.amazonaws.auth.BasicAWSCredentials
+import com.amazonaws.services.s3.{AmazonS3URI, AmazonS3Client}
+import com.amazonaws.services.s3.model.BucketLifecycleConfiguration
+import org.apache.spark.Logging
```

Comment:
We should really remove our dependence on Spark's `Logging` trait; I'll do this in a followup and will add a Scalastyle check to make sure it's not re-introduced.